NuMPI/muFFTnp/NetCDF.py

```diff
@@ -0,0 +1,503 @@
+import numbers
+from functools import reduce
+from math import sqrt
+
+import numpy as np
+
+from netCDF4 import Dataset
+
+from .Communicator import Communicator
+
+###
+
+class NCStructuredGridFrame(object):
```

Comment:
@mishramlan why do you include that ? 

I expect you don't need it 

And actually I think we have our own implementation of NCStructuredGrid in NuMPI. 

I am not even sure this part of the muFFT code works. There was a reason for us to implement our own.